solutions/project/hub-env/samples/Egress-workload-to-internet.md

```diff
@@ -12,8 +13,10 @@
         next
     end
     ```
+
 1. Web Filter profile allowing the required whitelist
-    - urlfilter: only allow access to packages.cloud.google.com, *.ubuntu.com, *.debian.org
+    - urlfilter: only allow access to packages.cloud.google.com, *.ubuntu.com,*.debian.org
```

Comment:
very minor - can leave as is - with space
*.ubuntu.com, *.debian.org

Reply:
Very good work Daniel
Cursory review good
I will exercise the readme/docs shortly (hopefully over the next week) - in a clean org kcc install via
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/issues/296